getting-started/macos/macos-basic-setup.md

```diff
@@ -0,0 +1,78 @@
+---
+ID:
+post_title: Basic Roots development environment for macOS
+author: Michael W. Delaney
+post_excerpt: ""
+layout: doc
+permalink: >
+
+published: false
+post_date:
+---
+
+Basic macOS 10.13+ development environment for Roots Trellis, Bedrock, and Sage
+
+## Requirements
+
+Roots Trellis, Bedrock, and Sage development on macOS is supported by several libraries and software packages. The following guide presumes each of these recommended packages are installed and configured on your computer. Recommended packages are:
+
+- Xcode Command Line Tools
```

Comment:
can we ideally link to all these?

Reply:
The language should actually be fixed because we don't presume these are installed; we are showing you how to install them. I'll rewrite the paragraph to be clearer.